### DISCUSSION TITLE
Show full error in dead/retries queue?

### DISCUSSION THREAD

**Discussion by andrewdang17** [sidekiq/sidekiq#6557] [Ideas]
it would be great if sidekiq UI showed the full error on hover or can expand somehow. It's currently truncated and not very helpful when the job keeps going in/out of retries

![Screenshot 2024-12-19 at 9 18 06 AM](https://github.com/user-attachments/assets/4cead4f4-760c-46b5-8601-ecedbe0eaf42)

**Comment by mperham** [comment]
You can see the full error by clicking into the job details. Is that sufficient?

**Reply by andrewdang17** [reply]
oh jeez I didn't realize that was what the `last retry` column was linking too 🤦  

thanks!
